lib/core/network/router.js

```diff
@@ -138,6 +138,36 @@ class Router {
         this._executeFromHttp(route.verb, request, cb);
       });
     }
+
+    /**
+     * Returns inner metrics from the router
+     * @returns {Object}
+     */
+    global.kuzzle.onAsk(
+      'core:network:router:metrics',
+      () => this.metrics());
+  }
+
+  /**
+   * Returns the metrics of the router
+   * @returns {Object}
+   */
+  metrics () {
+    const connectionsByProtocol = {};
+
+    for (const connection of this.connections.values()) {
+      const protocol = connection.connection.protocol.toLowerCase();
+      if (protocol !== 'internal') {
+        if (connectionsByProtocol[protocol] === undefined) {
+          connectionsByProtocol[protocol] = 0;
+        }
+        connectionsByProtocol[protocol]++;
+      }
```

Comment:
Nitpicking: you could use a guard instead to avoid indenting the whole code block
```js
      if (protocol === 'internal') {
        continue;
      }
      // rest of the code
```

Reply:
Yes good idea